tasks/rh-sign-image-cosign/rh-sign-image-cosign.yaml

```diff
@@ -25,11 +25,11 @@ spec:
     - name: sign-image
       image: quay.io/konflux-ci/release-service-utils:38c3bfd7479c86b832cba5b61f9bbde40c469393
       env:
-        - name: AWS_REGION
+        - name: AWS_DEFAULT_REGION
```

Comment:
Why this change? Was that a mistake?

Reply:
Oh, you already mention it in the commit message, my bad. (But not in the PR description, so it could be added there as well.

Reply:
Added it to description.

Reply:
Thanks.